<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatGUIClass</class>
 <widget class="QMainWindow" name="chatGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chatGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="mes">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBut">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>570</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Message:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>581</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listMessages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>599</width>
      <height>349</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>617</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="chatgui.qrc"/>
 </resources>
 <connections/>
</ui>
